src/store/sagas/index.js

```diff
@@ -54,9 +54,10 @@ export function* workMessagePolling({ data }) {
     });
     yield all(completed.map(paymentId => setCompleted(paymentId)));
     const maxIdentifier = findMaxMsgInternalId(data);
-
-    yield put({ type: SET_LATEST_INTERNAL_MSG_ID, id: maxIdentifier });
-
+    if (data && data.length && data.length >= 0) {
```

Comment:
I don't understand the last term of the conjunction. Isn't data.length always non-negative?

Reply:
don't understand your question

Reply:
we are checking if data.length is at least 1 or more

Reply:
you are checking if data.length is 0 or more

Reply:
true i will fix that